ckanext/fjelltopp_theme/templates/home/snippets/promoted.html

```diff
@@ -0,0 +1,23 @@
+{% block promote %}
+<div class="promoted promoted-home col-xs-12 col-sm-12 col-lg-12" >
+    {% block backgroundvideo %}
+    <video autoplay muted loop id="background-video">
+    <source src="/images/fjelltopp-bgs/fjelltopp.mp4" type="video/mp4">
```

Comment:
This MP4 file is 70% of the pages size and on slow 3G takes 2 minutes to transfer on slow 3G. Can we convert this to an animated .webp or something?